Reply/gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-accompanist = "0.25.1"
+accompanist = "0.26.2-beta"
```

Comment:
Don't edit this file directly. If you rebase you will see a comment in there for that. It will be overriden during the next update. This file is generated from a central file that manages the versions for all samples.

You might have to manually override the version in your build.gradle similar to how you are doing it with m3, or do this on the compose-latest branch.

Reply:
Got it 👌